docs/doc.md

```diff
@@ -70,7 +70,7 @@
 
 ### Build with json replacement
 
-Gin uses `encoding/json` as default json package but you can change it by build from other tags.
+Gin uses `encoding/json` as the default JSON package but you can change it by build from other tags.
```

Comment:
building

Reply:
Oh crap I completely missed that lol, thanks!